src/app/pages/public/ListLives.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useGetListLivesQuery } from 'C:/Users/Darren/Dropbox/PC/Desktop/code-appvantage/starter-kit/src/app/api/index';
```

Comment:
this import should be relative to its path, if someone clones your changes they were not be able to run it since it is using your local directory

Reply:
Apologies, I have changed to relative path instead